src/Chr.Avro/Resolution/ReflectionResolver.cs

```diff
@@ -466,7 +489,18 @@ public override TypeResolution Resolve(Type type)
                 throw new UnsupportedTypeException(type);
             }
 
-            return new ArrayResolution(type, itemType);
+            ICollection<ConstructorResolution> constructors = null;
```

Comment:
does `GetConstructors` blow up for array types? if not, would be cleaner to remove the conditional (and `Constructors` is never null on `ArrayResolution` , so null has no semantic meaning)

Reply:
Arrays have a strange constructor with parameters that don't have a name. Was cleaner to just ignore constructors for arrays then to add a where statement to filter out the strange constructor since we don't need a constructor for an Array.